internal/widget/base.go

```diff
@@ -8,115 +8,153 @@ import (
 	"fyne.io/fyne/v2/internal/cache"
 )
 
-// Base is a simple base widget for internal use.
+// Base provides a helper that handles basic widget behaviours.
 type Base struct {
-	hidden bool
-	pos    fyne.Position
-	size   fyne.Size
+	size     fyne.Size
+	position fyne.Position
+	hidden   bool
```

Comment:
FMI: why did you remove the alphabetical order?

Reply:
A mistake, sorry